src/index.js

```diff
@@ -14,12 +15,28 @@ function serializeComponents (props) {
 
     if (props[component].constructor === Function) { return; }
 
+    var ind = Object.keys(components).indexOf(component);
+    // Discards props that aren't components.
+    if (ind === -1) { return; }
+
     if (props[component].constructor === Array) {
       //Stringify components passed as array.
       serialProps[component] = props[component].join(' ');
     } else if (props[component].constructor === Object) {
       // Stringify components passed as object.
       serialProps[component] = styleParser.stringify(props[component]);
+    } else if (props[component].constructor === Boolean) {
+      if (components[component].schema.type === 'boolean') {
```

Comment:
Hard to wrap my brain around this, though I kinda get it. Can you add a description to the PR that details the logic?